tests/tensor_base/test_augmented_updates.py

```diff
@@ -0,0 +1,111 @@
+from numpy.testing import assert_allclose
+import mygrad as mg
+import numpy as np
+
+
+# TODO: These tests need to be modified to check that tensor-identity
+# is preserved by the augmented update once #280 is resolved
+# https://github.com/rsokl/MyGrad/pull/280/
+def test_augmented_multiply():
+    a = np.arange(9.0).reshape(3, 3)
+    expected_grad = np.where(a < 4, 2, 1)
+    a[a < 4] *= 2
+
+    t1 = mg.arange(9.0).reshape(3, 3)
+    y = mg.full(t1[t1 < 4].shape, 2.0)
+    t2 = +t1
```

Comment:
Probably worth adding a comment about how you want this to form a new node. I don't think that's idiomatic for anyone but you.

Reply:
Good call. Thanks!